.zenodo.json

```diff
@@ -1,382 +1,350 @@
+    },
+    {
+      "affiliation": "NIBIO, Norway",
+      "name": "Mohr, Christian Wilhelm",
+      "orcid": "0000-0003-2656-1802"
+    },
+    {
+      "affiliation": "University of Arizona, USA",
+      "name": "Amarjiit, Pandde"
+    },
+    {
+      "affiliation": "BSC, Spain",
+      "name": "Pérez-Zanón, Núria"
+    },
+    {
+      "affiliation": "NCAR, USA",
+      "name": "Phillips, Adam",
+      "orcid": "0000-0003-4859-8585"
+    },
+    {
+      "affiliation": "University of Arizona, USA",
+      "name": "Russell, Joellen"
+    },
+    {
+      "affiliation": "CICERO, Norway",
+      "name": "Sandstad, Marit"
+    },
+    {
+      "affiliation": "MetOffice, UK",
+      "name": "Sellar, Alistair"
+    },
+    {
+      "affiliation": "DLR, Germany",
+      "name": "Senftleben, Daniel"
+    },
+    {
+      "affiliation": "ISMAR-CNR, Italy",
+      "name": "Serva, Federico",
+      "orcid": "0000-0002-7118-0817"
+    },
+    {
+      "affiliation": "CICERO, Norway",
+      "name": "Sillmann, Jana"
+    },
+    {
+      "affiliation": "MPI-M, Germany",
+      "name": "Stacke, Tobias",
+      "orcid": "0000-0003-4637-5337"
+    },
+    {
+      "affiliation": "URead, UK",
+      "name": "Swaminathan, Ranjini",
+      "orcid": "0000-0001-5853-2673"
+    },
+    {
+      "affiliation": "BSC, Spain",
+      "name": "Torralba, Verónica"
+    },
+    {
+      "affiliation": "University of Bremen, Germany",
+      "name": "Weigel, Katja",
+      "orcid": "0000-0001-6133-7801"
+    },
+    {
+      "affiliation": "University of Reading, UK",
+      "name": "Roberts, Charles",
+      "orcid": "0000-0002-1147-8961"
+    },
+    {
+      "affiliation": "Netherlands eScience Center",
+      "name": "Kalverla, Peter",
+      "orcid": "0000-0002-5025-7862"
+    },
+    {
+      "affiliation": "Netherlands eScience Center",
+      "name": "Alidoost, Sarah",
+      "orcid": "0000-0001-8407-6472"
+    },
+    {
+      "affiliation": "Netherlands eScience Center",
+      "name": "Verhoeven, Stefan",
+      "orcid": "0000-0002-5821-2060"
+    },
+    {
+      "affiliation": "Netherlands eScience Center",
+      "name": "Vreede, Barbara",
+      "orcid": "0000-0002-5023-4601"
+    },
+    {
+      "affiliation": "Netherlands eScience Center",
+      "name": "Smeets, Stef",
+      "orcid": "0000-0002-5413-9038"
+    },
+    {
+      "affiliation": "Netherlands eScience Center",
+      "name": "Soares Siqueira, Abel",
+      "orcid": "0000-0003-4451-281X"
+    },
+    {
+      "affiliation": "DLR, Germany",
+      "name": "Kazeroni, Rémi",
+      "orcid": "0000-0001-7205-9528"
+    },
+    {
+      "affiliation": "NASA, USA",
+      "name": "Potter, Jerry"
+    }
+  ],
+  "description": "ESMValTool: A community diagnostic and performance metrics tool for routine evaluation of Earth system models in CMIP.",
+  "license": {
+    "id": "Apache-2.0"
+  },
+  "publication_date": "2022-07-25",
+  "title": "ESMValTool",
+  "version": "v2.6.0"
```

Comment:
I wouldn't know, Remi! Maybe @glpotter knows?

Reply:
My point here is that running `cffconvert --format zenodo --outfile .zenodo.json` deletes the info about "communities" and "grants" stored at the end of the `.zenodo.json` file, starting from [here](https://github.com/ESMValGroup/ESMValTool/blob/70f978e13279cea1494a58fd7c5e78377f75c666/.zenodo.json#L348). Is that the expected behaviour? I understand the suggested usage of the `cffconvert` command was recently changed and the output file `.zenodo.json` file was not updated accordingly. I wonder if it is fine to delete the info about "communities" and "grants"... Anyway, that should not be related to the CMORizer update done in this PR.

Reply:
We have a remark about this in the ESMValCore documentation, but unfortunately, it is missing in ESMValTool documentation: https://docs.esmvaltool.org/projects/esmvalcore/en/latest/contributing.html#list-of-authors. The "grants" and "communities" should not be removed, they are shown on Zenodo because they are in this file. You can see them in the column on the right here: https://doi.org/10.5281/zenodo.3401363.

Reply:
gotcha! Thanks, Bouwe, we don't remove them then , wouldn't want to upset the cash cows :cow2:

Reply:
also @bouweandela note the changes in calling `cffconvert` from https://github.com/ESMValGroup/ESMValTool/pull/2791/files